extensions/amp-slike/validator-amp-slike.protoascii

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-slike
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-slike"
+    version: "0.1"
+    version: "latest"
+    requires_usage: EXEMPTED
```

Comment:
I would recommend ERROR here. EXEMPTED is typically reserved for legacy extensions: https://github.com/ampproject/amphtml/blob/91aaf6f3e28c28117fc6ca0eb660d39161e387ee/validator/validator.proto#L496-L498

Alternatively, it is possible to use NONE, if there is some useful non-nil behavior when the `<script>` is on the page but not the `<amp-slike>`, but from a quick glance at the docs & code that doesn't appear to be the case.

Reply:
I've since learned you can just delete this line since ERROR is the default value.